drivers/tee/optee/Kconfig

```diff
@@ -37,6 +37,20 @@ config OPTEE_TA_SCP03
 	help
 	  Enables support for controlling (enabling, provisioning) the
 	  Secure Channel Protocol 03 operation in the OP-TEE SCP03 TA.
+
+config TEE_I2C_NXP_SE05X_ERRATA
+	bool "Enable NXP SE05X Errata"
+       select TEE_I2C_NXP_SE05X_ERRATA_IN_BUS
+	default y
```

Comment:
IMHO this should be disabled by default.

Reply:
if this is active is because there is a secure element
And if there is a secure element, it will be an NXP SE050
And if there is an NXP SE050 it will very likely be affected by the errata (today and for the next couple of years I think)

but on top of that we shouldnt ping the device on every single transfer (I always thought that it was a waste but upstream didnt get the point and I didnt have the time - or interest- to argue ).....

The "errata"  achieves the same thing with less penalties on the bus so I'd much rather keep this implementation by default.

Reply:
I think it is OK to have it enabled by default as long it depends on SE050.

Reply:
AFAIK there is no other trampoline user....